README.md

```diff
@@ -1407,6 +1408,8 @@ knpu_oauth2_client:
         # will create service: "knpu.oauth2.client.twitch"
         # an instance of: KnpU\OAuth2ClientBundle\Client\Provider\TwitchClient
         # composer require depotwarehouse/oauth2-twitch
+        # This is deprecated and won't be supported by Twitch in the near future (Feb. 28 2022)
+        # use the twitch_helix one (directly below) if you create a new app
```

Comment:
This is auto-generated section. Please, run `bin/update_readme` instead to update this correctly.

Also, if the old Twitch way is deprecated, please, update its name in `TwitchProviderConfigurator::getProviderDisplayName()` to `Twitch (deprecated)`